Sat May 28 16:26:12 2022
Darwin-21.1.0-x86_64-i386-64bit
Ren'Py 7.4.11.2266
 

Bootstrap to the start of init.init took 0.08s
Early init took 0.00s
Loader init took 0.01s
Loading error handling took 0.04s
Loading script took 0.84s
Loading save slot metadata. took 0.01s
Loading persistent took 0.00s

File "game/script.rpy", line 1104: screen expects a non-empty block.
    screen puzzle2blocks():
                           ^
DPI scale factor: 1.000000
Interface start took 0.86s

Initializing gl2 renderer:
primary display bounds: (0, 0, 1440, 900)
swap interval: 1 frames
Windowed mode.
Vendor: 'Apple'
Renderer: 'Apple M1'
Version: '2.1 Metal - 76.1'
Display Info: None
Screen sizes: virtual=(1920, 1080) physical=(1239, 697) drawable=(2478, 1394)
Maximum texture size: 4096x4096
Resetting cache.
Bootstrap to the start of init.init took 12.12s
Early init took 0.00s
Loader init took 0.01s
Loading error handling took 0.03s
game/script.rpy:1881: SyntaxWarning: name 'i_overlap' is used prior to global declaration
  global i_overlap
game/script.rpy:1882: SyntaxWarning: name 'ie_overlap' is used prior to global declaration
  global ie_overlap
Loading script took 0.83s
Loading save slot metadata. took 0.01s
Loading persistent took 0.00s
Importing _renpysteam: ImportError('No module named _renpysteam',)

Full traceback:
  File "game/script.rpy", line 2, in script
    python:
  File "/Applications/renpy-7.4.11-sdk/renpy/ast.py", line 928, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "/Applications/renpy-7.4.11-sdk/renpy/python.py", line 2245, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/script.rpy", line 4, in <module>
    import gspread
  File "/Applications/renpy-7.4.11-sdk/lib/python2.7/gspread/__init__.py", line 16, in <module>
    from .auth import oauth, oauth_from_dict, service_account, service_account_from_dict
  File "/Applications/renpy-7.4.11-sdk/lib/python2.7/gspread/auth.py", line 12, in <module>
    from google.oauth2.credentials import Credentials
ImportError: No module named google.oauth2.credentials

While running game code:
  File "game/script.rpy", line 2, in script
    python:
  File "game/script.rpy", line 4, in <module>
    import gspread
ImportError: No module named google.oauth2.credentials
DPI scale factor: 1.000000

Full traceback:
  File "/Applications/renpy-7.4.11-sdk/renpy/bootstrap.py", line 331, in bootstrap
    renpy.main.main()
  File "/Applications/renpy-7.4.11-sdk/renpy/main.py", line 560, in main
    renpy.game.context().run(node)
  File "game/script.rpy", line 2, in script
    python:
  File "game/script.rpy", line 2, in script
    python:
  File "/Applications/renpy-7.4.11-sdk/renpy/ast.py", line 928, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "/Applications/renpy-7.4.11-sdk/renpy/python.py", line 2245, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/script.rpy", line 4, in <module>
    import gspread
  File "/Applications/renpy-7.4.11-sdk/lib/python2.7/gspread/__init__.py", line 16, in <module>
    from .auth import oauth, oauth_from_dict, service_account, service_account_from_dict
  File "/Applications/renpy-7.4.11-sdk/lib/python2.7/gspread/auth.py", line 12, in <module>
    from google.oauth2.credentials import Credentials
ImportError: No module named google.oauth2.credentials

While running game code:
  File "game/script.rpy", line 2, in script
    python:
  File "game/script.rpy", line 2, in script
    python:
  File "game/script.rpy", line 4, in <module>
    import gspread
ImportError: No module named google.oauth2.credentials
